### PR TITLE
Incorrect City Spelling in uk_UA locale

### PR DESCRIPTION
**Pull request by ch4zzy**
### What does this change

The names of the following cities have been changed:

- Zaporizhia
- Odessa
- Khmelnytsky
- Kiev
- Uzhgorod
- Zaporozhye

### What was wrong

The current city names are not responding internationally recognized spelling.
[Wiki list of cities in Ukraine with correct spelling.](https://en.wikipedia.org/wiki/List_of_cities_in_Ukraine)
[Wiki list of administrative divisions in Ukraine with correct spelling.](https://en.wikipedia.org/wiki/Administrative_divisions_of_Ukraine)

### How this fixes it

- Zaporizhia -> Zaporizhzhia
- Odessa -> Odesa
- Khmelnytsky -> Khmelnytskyi
- Kiev -> Kyiv
- Uzhgorod -> Uzhhorod
- Zaporozhye -> Zaporizhzhia

### CODE REVIEW

File: faker/providers/date_time/__init__.py [hunk 1]
```diff
@@ -1426,16 +1426,16 @@ def _rand_seconds(self, start_datetime: int, end_datetime: int) -> float:
         ),
         Country(
             timezones=[
-                "Europe/Kiev",
-                "Europe/Uzhgorod",
-                "Europe/Zaporozhye",
+                "Europe/Kyiv",
+                "Europe/Uzhhorod",
```
Comment (fcurella) [inline faker/providers/date_time/__init__.py:1430]: According to the [tz database](https://en.wikipedia.org/wiki/List_of_tz_database_time_zones), this should still be `"Europe/Uzhgorod"`

File: faker/providers/date_time/__init__.py [hunk 2]
```diff
@@ -1426,16 +1426,16 @@ def _rand_seconds(self, start_datetime: int, end_datetime: int) -> float:
...
+                "Europe/Zaporizhzhia",
```
Comment (fcurella) [inline faker/providers/date_time/__init__.py:1431]: Similarly, this should still be `"Europe/Zaporozhye"`

**Review by fcurella — CHANGES_REQUESTED** [review]

File: faker/providers/date_time/__init__.py (hunk 2 above)
Comment (ch4zzy) [reply faker/providers/date_time/__init__.py:1431]: I agree, I initially misunderstood the meanings.

File: faker/providers/date_time/__init__.py (hunk 1 above)
Comment (fcurella) [reply faker/providers/date_time/__init__.py:1430]: Could you change this back to `"Europe/Uzhgorod"`?

File: faker/providers/date_time/__init__.py (hunk 2 above)
Comment (fcurella) [reply faker/providers/date_time/__init__.py:1431]: This should be changed back to `"Europe/Zaporozhye"`

**Review by fcurella — CHANGES_REQUESTED** [review]

**Review by fcurella — APPROVED** [review]
Thank you!

**Comment by ghostishev** [comment]
Thank you!

**Pull request merged**